src/lib.rs

```diff
@@ -35,11 +25,11 @@ pub type LogicalOutputPluginInit =
                               -> ()>;
 */
 
-extern fn startup(ctx: *mut libpq::Struct_LogicalDecodingContext,
+unsafe extern fn startup(ctx: *mut libpq::Struct_LogicalDecodingContext,
```

Comment:
I don't think this works here: `unsafe extern` and `extern` are not the same; so if the earlier types work, I'm not sure how these can. Does it have to do with changes in bindgen?